pkg/relay/ethereum/events.go

```diff
@@ -79,7 +82,8 @@ func (chain *Chain) findSentPackets(ctx core.QueryContext, fromHeight uint64) (c
 
 		sendPacket, err := chain.ibcHandler.ParseSendPacket(log)
 		if err != nil {
-			return nil, fmt.Errorf("failed to parse SendPacket event: err=%v, log=%v", err, log)
+			revertReason, data := chain.parseRpcError(err)
+			return nil, fmt.Errorf("failed to parse SendPacket event: err=%v, log=%v, reason=%s, data=%s", err, log, revertReason, data)
```

Comment:
`ParseSendPacket` is not a RPC call, right?

Reply:
yes. fixed.